cli/src/diff_util.rs

```diff
@@ -520,8 +520,13 @@ impl ColorWordsDiffOptions {
                 })?),
             }
         };
+        let context = if let Some(c) = args.context {
+            c
+        } else {
+            config.get::<usize>("diff.color-words.context")?
```

Comment:
nit: `::<usize>` can be omitted. (I would add `let context: usize` if I need to make the type clear.)

We can also use `map_or_else()`, but I don't have any preference about that.
https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.map_or_else

Reply:
What an interesting pattern, thanks for pointing it out.